.eslintignore

```diff
@@ -1,4 +1,10 @@
 node_modules
 test/temp
 output
-__transpiled
\ No newline at end of file
+__transpiled
```

Comment:
You don't need these 3 files :) `test/temp`, `output` and `__transpiled` is only used in some templates.